src/ott/solvers/linear/sinkhorn.py

```diff
@@ -409,7 +410,7 @@ def converged(self) -> bool:  # noqa: D102
     if self.errors is None:
       return False
     return jnp.logical_and(
-        jnp.any(self.errors == -1), jnp.all(jnp.isfinite(self.errors))
+        not jnp.any(jnp.isnan(self.errors)), self.errors[-1] < self.threshold
```

Comment:
I think you need to use `jnp.logical_not`

Reply:
Thanks!!